sunpy/map/sources/psp.py

```diff
@@ -0,0 +1,43 @@
+"""
+Solar Orbiter Map subclass definitions.
```

Comment:
Check here. This should be: **Parker Solar Probe Map subclass definitions.**  right?